ibid/plugins/ddg.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2008-2011, Michael Gorven, Stefano Rivera
+# Released under terms of the MIT/X/Expat Licence. See COPYING for details.
+
+from httplib import BadStatusLine
+
+from ibid.config import Option
+from ibid.plugins import Processor, match
+from ibid.utils import decode_htmlentities, json_webservice
+
+features = {'duckduckgo': {
+    'description': u'Retrieves results from DuckDuckGo',
+    'categories': ('lookup', 'web', 'calculate', ),
+}}
+
+default_user_agent = 'Mozilla/5.0'
+default_referer = "http://ibid.omnia.za.net/"
```

Comment:
Stick to one quote style for the whole file. Single quotes tend to be preferred.